src/node/miner.h

```diff
@@ -160,6 +160,17 @@ class BlockAssembler
         // Whether to call TestBlockValidity() at the end of CreateNewBlock().
         bool test_block_validity{true};
         bool print_modified_fee{DEFAULT_PRINT_MODIFIED_FEE};
+        /**
+         * The maximum additional weight which the pool will add to the coinbase
+         * scriptSig, witness and outputs. This must include any additional
+         * weight needed for larger CompactSize encoded lengths.
+         */
```

Comment:
Indicate the assumed upper limits of this options?

Reply:
#29432 implements this in more detail. I'm not even 100% sure if it's correct there, so I'd rather wait with documentation the exact numbers - that might just encourage someone to recompile and burn themselves.